go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/hashicorp/vault-plugin-secrets-gcp
 
-go 1.12
+go 1.13
+
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Comment:
At least in the main repo we've pulled this out:  https://github.com/hashicorp/vault/commit/fa245e6a1db4bd8b8328c6c1a9850269078a5da9

Reply:
Oh nice. I'm not gonna question why it's suddenly fixed.